src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -397,8 +403,13 @@ def load_arguments(self, _):
                    options_list=['--name', '-n', '--resource-group', '-g'], local_context_attribute=None)
 
     with self.argument_context('tag') as c:
-        c.argument('tag_name', options_list=['--name', '-n'])
-        c.argument('tag_value', options_list='--value')
+        c.argument('tag_name', tag_name_type)
+        c.argument('tag_value', tag_value_type)
+        c.argument('resource_id', tag_resource_id_type)
+        c.argument('tags', tags_type)
+        c.argument('operation', options_list='--operation',
```

Comment:
options_list for 'operation' is unnecessary.

Reply:
Removed